bs_detector/bias.py

```diff
@@ -475,27 +483,59 @@ def print_raw_data_for_features(list_of_sentences):
             print(feat.keys())
             KEYS_DONE = True
         print(feat.values())
-#print_raw_data_for_features(get_list_from_file('input_text_original'))
+
+
+# print_raw_data_for_features(get_list_from_file('input_text_original'))
```

Comment:
this dead code can be deleted

Reply:
Ask @cjhutto about it. My IDE fixed some spacing with the comment sign is all that change was.

Reply:
it was already deleted on the most recent update (where I added html output and TSV output formats)

Reply:
Ah ok, I may have pulled right before your latest changes. Will update :)

Reply:
Ok then it will get deleted when merge conflicts are handled.